MM2RandoHost/MainWindow.xaml

```diff
@@ -10,6 +10,38 @@
         Title="MM2 Randomizer" Height="370" Width="403.259" Drop="Window_Drop" DragEnter="Window_DragEnter" DragLeave="Window_DragLeave">
     <Window.Resources>
         <converters:InverseBooleanConverter x:Key="inverseBooleanConverter"/>
+        <Style x:Key="placeHolder" TargetType="{x:Type TextBox}" BasedOn="{StaticResource {x:Type TextBox}}">
```

Comment:
I like the custom style, except the name "placeHolder". Even better, styles should be defined in an external ResourceDictionary (perhaps in a Resources folder in the project)